templates/takeovers/_multicloud-operations.html

```diff
@@ -0,0 +1,77 @@
+<section class="p-strip--image is-deep js-takeover is-bordered p-takeover--multicloud-operations">
```

Comment:
Please add as another class: ` {% if not nojs_fallback %}u-hide{% endif %}` 

(see `takeovers/_tackling_iot.html` for example)